src/runtime/exec_context.ts

```diff
@@ -283,7 +295,7 @@ export default class ExecContext<M> {
   public destroy() {
     this.stateManager().stop(
       this,
-      this.subscriptions(this.state()),
+      this.subscriptions(this.state() as any as M),
```

Comment:
`as any as M` ???

Reply:
Yeah, had to force it to `M` because it thinks it's a generic `object`. Something for a future pass.